tooling/debugger/src/lib.rs

```diff
@@ -74,25 +79,77 @@ impl<'backend, B: BlackBoxFunctionSolver> DebugContext<'backend, B> {
             println!("Finished execution");
         } else {
             println!("Stopped at opcode {}: {}", ip, opcodes[ip]);
-            Self::show_source_code_location(&OpcodeLocation::Acir(ip), &self.debug_artifact);
+            self.show_source_code_location(&OpcodeLocation::Acir(ip), &self.debug_artifact);
         }
     }
 
-    fn show_source_code_location(location: &OpcodeLocation, debug_artifact: &DebugArtifact) {
+    fn print_location_path(&self, loc: Location) {
+        let line_number = self.debug_artifact.location_line_number(loc).unwrap();
+        let column_number = self.debug_artifact.location_column_number(loc).unwrap();
+
+        println!(
+            "At {}:{line_number}:{column_number}",
+            Files::name(&self.debug_artifact, loc.file).unwrap()
+        );
+    }
+
+    fn show_source_code_location(&self, location: &OpcodeLocation, debug_artifact: &DebugArtifact) {
         let locations = debug_artifact.debug_symbols[0].opcode_location(location);
         if let Some(locations) = locations {
```

Comment:
```suggestion
        let Some(locations) = locations else { return }
```

This will help remove some indentation.

Reply:
Neat 👌